modules/messages/client/components/UnreadCount.component.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { watch as watchUnreadCount } from '@/modules/messages/client/services/unread-message-count.client.service';
+
+export default function UnreadCount() {
+  const { t } = useTranslation('messages');
+
+  const [count, setCount] = useState(null);
+
+  useEffect(() => watchUnreadCount(setCount), []);
+
+  if (count === null || count === 0) {
```

Comment:
Yes, that was my desire, because of how my brain works, but you're right, there is no practical reason to distinguish them. I'll change it to default to 0. If we did need to distinguish them in the future it would not be hard.

Reply:
@nicksellen I'd say, stick with whatever makes more sense to you. Don't switch if you like it the way it is. These are just opinions, and you're the author.

These things don't matter to me, either way, really. I guess I just had nothing else to comment to, yesterday. :slightly_smiling_face: